asserts/snap_asserts.go

```diff
@@ -77,6 +83,8 @@ func (snapdcl *SnapDeclaration) checkConsistency(db RODatabase, acck *AccountKey
 	if err != nil {
 		return err
 	}
+
+	// FIXME check that snap-ids in refresh-control are valid?
```

Comment:
Yes, while it would be nice to be able to make sure data is sane, considering cross-gating snaps, for example, this is probably not worth it.